regression/tests/lms/test_discussions.py

```diff
@@ -1,47 +1,104 @@
-import os
+import time
+import random
+import unittest
+from uuid import uuid4
 
 from bok_choy.web_app_test import WebAppTest
+from bok_choy.page_object import WrongPageError
 from bok_choy.page_object import PageLoadError
 
-from regression.pages.lms import LMS_STAGE_BASE_URL, LMS_PROTOCOL
-from regression.tests.helpers.api_clients import LmsLoginApi
 from edxapp_acceptance.pages.lms.discussion import DiscussionThreadPage
 
+from regression.pages.lms.discussions_page import DiscussionsHomePageExtended
+from regression.pages.lms.discussions_page import DiscussionThreadPageExtended
+from regression.tests.helpers.api_clients import LmsLoginApi
+from regression.tests.helpers.utils import get_course_info
+from regression.pages.lms.utils import get_course_key
+
 
 class DiscussionTest(WebAppTest):
     """
     Test to check that discussion page is available
     """
-    def test_course_discussion_page(self):
-        LMS_BASE_URL = os.environ.get('LMS_BASE_URL', LMS_STAGE_BASE_URL)
-        COURSE_ORG = os.environ.get('COURSE_ORG', 'edX')
-        COURSE_NUMBER = os.environ.get('COURSE_NUMBER', 'DemoX')
-        COURSE_RUN = os.environ.get('COURSE_RUN', 'Demo_Course')
+
+    def setUp(self):
+        super(DiscussionTest, self).setUp()
 
         login_api = LmsLoginApi()
         login_api.authenticate(self.browser)
-        discussion_page = DiscussionThreadPage(self.browser, '.page-content')
+        self.discussion_thread_page = DiscussionThreadPage(self.browser, '.page-content')
+
+        self.discussion_home_page_ext = DiscussionsHomePageExtended(self.browser, get_course_key(get_course_info()))
 
-        discussion_page.url = '{}://{}/courses/course-v1:{}+{}+{}/discussion/forum/'.format(
-            LMS_PROTOCOL,
-            LMS_BASE_URL,
-            COURSE_ORG,
-            COURSE_NUMBER,
-            COURSE_RUN
-        )
+        self.post_title = 'post_to_test{}'.format(uuid4().hex)
```

Comment:
Is it intent to have different values for all post fields?
So now we have different tail for each field.